attachments/validators.py

```diff
@@ -0,0 +1,34 @@
+import magic
+from django.core.exceptions import ValidationError
+import clamd
+from django import forms
+from django.conf import settings
+
+# Validate file type using python magic and
+# scan for virus using python clamd
+
+
+def validate_file_type(upload):
+    upload.seek(0)
+
+    # Get File type
+    file_type = magic.from_buffer(upload.read(1024), mime=True)
+
+    # Check if file type satisfies our types
+    if hasattr(settings, 'ALLOWED_FILE_TYPES') and \
+        file_type not in settings.ALLOWED_FILE_TYPES:
```

Comment:
I made some changes to this block of code in bitbucket to allow the list of allowed filetypes to be dynamic (not jut the mime type) - so I changed the value into a dictionary, not just a list. - please see the latest and make the changes here as well.

Reply:
Sure!